docs/en/ingest-management/commands.asciidoc

```diff
@@ -210,6 +212,11 @@ verification.
 `--certificate-authorities <string>`::
 Comma-separated list of root certificates used for server verification.
 
+`--delay-enroll`::
+Delays enrollment to occur on first start of the {agent} service. This setting
+is useful when there is no ability to run {agent} at installation time, for
```

Comment:
```suggestion
is useful when the install doesn't want the {agent} to start until next reboot or manual start of the service, for
```

Reply:
Made a couple of minor edits to this statement to remove anthropomorphism (the install can't want things). Hope that's OK! :-)